src/inference/src/dev/core_impl_ie.cpp

```diff
@@ -240,7 +240,8 @@ std::map<std::string, std::string> ov::CoreImpl::GetSupportedConfig(const std::s
     }
     for (auto&& config : configs) {
         auto parsed = parseDeviceNameIntoConfig(config.first);
-        if (deviceName.find(parsed._deviceName) != std::string::npos) {
+        std::string devName = config.first.find("GPU") != std::string::npos ? config.first : parsed._deviceName;
```

Comment:
ov::core doesn't pass all the configurations to AUTO plugin?

Reply:
BTW, conditions for specific devices are prohibited in common code